nix/cargo-doc-live_test.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, ... }: {
+  services.cargo-doc-live."cargo-doc-live1" = {
+    enable = true;
+    crateName = "simple";
```

Comment:
But there's no crate named `simple` in this PR? This is probably the PR is failing.

Reply:
@srid well, I reckon the error is there because `cargo` is not there when running `cargo-watch`

```
✔ ~/services-flake/example/cargo-doc-live [add-cargo-doc-live|● 2✚ 1…2] 
08:45 $ cargo-watch watch -- cargo test
error: No such file or directory (os error 2)
✘-1 ~/services-flake/example/cargo-doc-live [add-cargo-doc-live|● 2✚ 1…2] 
08:45 $ which cargo
/usr/bin/which: no cargo in
```

after I introduced `cargo`, it worked as expected

```
✔ ~/services-flake/example/cargo-doc-live [add-cargo-doc-live|● 2✚ 1…2] 
08:46 $ cargo-watch watch -- cargo test
[Running 'cargo test']
    Finished `test` profile [unoptimized + debuginfo] target(s) in 0.02s
     Running unittests src/main.rs (target/debug/deps/simple-d0979b29c0218862)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

Reply:
@srid I made some changes to add `cargo` as one of inputs https://github.com/juspay/services-flake/pull/246/commits/109a6db3a76519a26186884aca60c763c171260c#diff-78d56bff6aefc6708f78b88189a646cb93da9701dc1c3bed9f553dbe90b2067dR33, now it's giving me a different error

```
cargo-doc-live-test> + process-compose --no-server -t=false
cargo-doc-live-test> [cargo-doc-live1-cargo-doc ] error: project root does not exist
```